root/static/scripts/edit/URLCleanup.js

```diff
@@ -3976,6 +3976,45 @@ const CLEANUPS: CleanupEntries = {
       return {result: false, target: ERROR_TARGETS.URL};
     },
   },
+  'tidal': {
+    match: [new RegExp(
+      '^(https?://)?' +
+      '(([^/]+\\.)*(desktop|listen|stage|store|www)\\.)?tidal\\.com' +
+      '/.*(album|artist|track|video)/',
+      'i',
+    )],
+    restrict: [LINK_TYPES.streamingpaid],
+    clean: function (url) {
+      url = url.replace(/^(?:https?:\/\/)?(?:(?:[^\/]+\.)*(?:desktop|listen|stage|store|www)\.)?tidal\.com\/(?:#!\/)?([\w\/]+).*$/, 'https://tidal.com/$1');
+      url = url.replace(/^https:\/\/tidal\.com\/(?:[a-z]{2}\/)?(?:browse\/|store\/)?(?:[a-z]+\/\d+\/)?([a-z]+)\/(\d+)(?:\/[\w]*)?$/, 'https://tidal.com/$1/$2');
+      return url;
+    },
+    validate: function (url, id) {
+      const m = /^https:\/\/tidal\.com\/([a-z]+)\/\d+$/.exec(url);
+      if (m) {
+        const prefix = m[1];
+        switch (id) {
+          case LINK_TYPES.streamingpaid.artist:
+            if (prefix === 'artist') {
+              return {result: true};
+            }
+            return {result: false, target: ERROR_TARGETS.ENTITY};
+          case LINK_TYPES.streamingpaid.release:
+            if (prefix === 'album') {
+              return {result: true};
+            }
+            return {result: false, target: ERROR_TARGETS.ENTITY};
+          case LINK_TYPES.streamingpaid.recording:
+            if (prefix === 'track' || prefix === 'video') {
```

Comment:
You said in https://github.com/metabrainz/musicbrainz-server/pull/2261#discussion_r710119259 that videos should be supported for releases, but here they are not. We should be consistent maybe? But more discussion there.

Reply:
Actually, I assumed the code was intentional and the commit message incomplete; See comment.
But yes, whatever is decided, it should be the same for videos here.

Reply:
I guess given that discussion videos should be unblocked for releases too?

Reply:
My bad, I forgot about it. Updated with commit <https://github.com/metabrainz/musicbrainz-server/pull/2263/commits/004f5b86f68bdb4c928b6b2dc4c673b9647fb6b5>.